src/app/workspace/resource/resource.component.ts

```diff
@@ -319,8 +321,11 @@ export class ResourceComponent implements OnInit, OnChanges, OnDestroy {
                             this.requestIncomingResources(this.resource);
                         }
 
-                        // gather resource property information
-                        res.resProps = this.initProps(response);
+                        setTimeout(() => {
```

Comment:
Why does this need to be wrapped in a `setTimeout`?

Reply:
Sorry, all of these you've find are leftovers I forgot to remove. Haven't work with TS lately and my IDE is not removing unused imports though.